src/example_configs/config_multSubs.json

```diff
@@ -0,0 +1,7 @@
+{
+    "dataOption": "live",
+    "madParameters": 10, 
+    "windowSize": 20,
+    "subscriptions": [{"prefix": "151.101.0.0/16"}, 
+                      {"host": "rrc21", "prefix": "0.0.0.0/0"}]
```

Comment:
similar to above, maybe changing this prefix to `{"path": "54113$"}` would be useful. Having a third IPv6 example would be neat: `{"prefix":"2a04:4e42:200::/48"}`